src/Pages/AddOrEditPostPage/index.tsx

```diff
@@ -2,20 +2,41 @@ import { useState } from 'react';
 import { useNavigate } from 'react-router-dom';
 import AddOrEditPostModal from '@/Components/Common/Modal/AddOrEditPostModal';
 import useTabStore from '@/Stores/Tab';
+import Alert from '@/Components/Common/Alert';
 
 const AddOrEditPostPage = () => {
   const [isModalOpen, setIsModalOpen] = useState(true);
   const { prev, setTab } = useTabStore();
-
+  const [alertMessage, setAlertMessage] = useState<string>('');
   const naviagte = useNavigate();
-  const handleCloseModal = (state: boolean) => {
+
+  const handleOpenAlert = () => {
+    setAlertMessage('정말로 나가시겠습니까?');
+  };
+
+  const handleCloseModal = () => {
     naviagte(-1);
-    setIsModalOpen(state);
+    setIsModalOpen(false);
     setTab(prev);
   };
 
+  const handleCloseAlert = () => {
+    setAlertMessage('');
+  };
+
   return isModalOpen ? (
-    <AddOrEditPostModal onChangeOpen={handleCloseModal} />
+    <>
+      {alertMessage && (
+        <Alert
```

Comment:
모바일 사이즈일 때는 조금 더 width 키워줘도 좋을 것 같아요! 저도 좀 키워가지고...
```suggestion
const { isMobileSize } = useResize();
        <Alert
        width={isMobileSize? 40 : undefined}
```
이렇게하면 비율 딱 괜찮은 것 같은데.. 어떠신가요??